modin/pandas/groupby.py

```diff
@@ -930,138 +1022,103 @@ def _wrap_aggregation(
         ----------
         qc_method : callable
             The query compiler method to call.
-        default_func : callable
-            The function to call if we need to default to pandas.
-        drop : bool, default: True
-            Whether or not to the grouping columns should be dropped on this operation.
-        numeric_only : bool, default: True
+        numeric_only : bool, default: None
```

Comment:
```suggestion
        numeric_only : {None, True, False}, default: None
```
plus augment the explanation of the variable? it's no longer a true boolean when it could be any of `True`, `False` and `None`...

Reply:
applied